tests/users/factories.py

```diff
@@ -189,7 +189,7 @@ class Params:
             ),
         )
         with_pole_emploi_id = factory.Trait(
-            jobseeker_profile__pole_emploi_id=factory.fuzzy.FuzzyText(length=8, chars=string.digits),
+            jobseeker_profile__pole_emploi_id=factory.fuzzy.FuzzyText(length=11, chars=string.digits),
```

Comment:
Pour continuer à tester les 2 formats ;)
```suggestion
            jobseeker_profile__pole_emploi_id=factory.Faker("numerify", "########!!!"),
```

Reply:
Je connaissais pas ce faker là :star_struck:

Reply:
Ah, ça ne marche pas : chaque `!` a une chance différente d'être Null ou un chiffre.
Donc on a des id de 8, 9, 10 ou 11 caractères

Reply:
Ah bah oui, pas encore le cerveau tout réveillé, désolé pour le bruit :(.